controllers/syncer/common.go

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2019 IBM Corp.
```

Comment:
can you change the year to 2022?

Reply:
is it on us to update?
what are the requirements for it?

Reply:
Changing

Reply:
> is it on us to update? what are the requirements for it?

for every new file here in operator we add the Copyright with the current year